app/pages/cif/project/[project]/index.tsx

```diff
@@ -4,24 +4,140 @@ import { graphql, usePreloadedQuery } from "react-relay/hooks";
 import { ProjectOverwiewQuery } from "__generated__/ProjectOverwiewQuery.graphql";
 import withRelayOptions from "lib/relay/withRelayOptions";
 
-const pageQuery = graphql`
+export const pageQuery = graphql`
   query ProjectOverwiewQuery($project: ID!) {
```

Comment:
I was going to fix the typo in `ProjectOverwiewQuery` but then it occured to me changing a query name might cause problems elsewhere?